tests/snmp/test_snmp_phy_entity.py

```diff
@@ -614,7 +614,7 @@ def test_turn_off_psu_and_check_psu_info(duthosts, enum_rand_one_per_hwsku_hostn
     pdu_controller.turn_off_outlet(first_outlet)
     assert wait_until(30, 5, check_outlet_status, pdu_controller, first_outlet, False)
     # wait for psud update the database
-    assert wait_until(120, 20, _check_psu_status_after_power_off, duthost, localhost, creds_all_duts)
+    assert wait_until(180, 20, _check_psu_status_after_power_off, duthost, localhost, creds_all_duts)
```

Comment:
@slutati1536 what is the motivation to increase timeout?

Reply:
hey @liat-grozovik, the dut took longer than 120 seconds to update the PSU status, but it passed consistently when the timeout was extended to 180 seconds